### ISSUE TITLE
⌨ `ViewPoolPage` draft status text has low contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#16597]
### ⌨ Accessibility and Expected Behaviour

The text for a draft pool does not meet colour contrast ratio requirements.

### 🕵️ Details

The text is using `text-warning-500` in light mode which is too light. We should be using `text-warning-600`.

https://61e099a1bb1465003a73d3ce-kyfpiytnvq.chromatic.com/?path=/story/pages-pools-viewpoolpage--draft-complete-process

### 📸 Screenshot

<img width="549" height="289" alt="Image" src="https://github.com/user-attachments/assets/b2159040-5295-44ff-a9f1-bf40da719976" />

### 🙋‍♀️ Proposed Solution

Update the draft status text to use `text-warning-600` on light mode. 

### ✅ Acceptance Criteria

- [ ] No colour contrast issues on the `ViewPoolPage`
